assets/javascripts/discourse/components/channels-list.js

```diff
@@ -78,6 +78,10 @@ export default class ChannelsList extends Component {
     }`;
   }
 
+  get showMobileDirectMessageButton() {
```

Comment:
can probably use https://api.emberjs.com/ember/release/functions/@ember%2Fobject%2Fcomputed/and

Reply:
I'm having troubles making use of `@and()`. It appears it doesn't support `this`
```javascript
@and("this.site.mobileView", "this.showDirectMessageChannels") showMobileDirectMessageButton;
```
results in:

```javascript
Uncaught (in promise) Error: Assertion Failed: 'this' in paths is not supported
```

Reply:
sis you try to just remove the this. part?

Reply:
🤦🏽‍♂️ Thought I tried that and got an error, but trying it again seems to work with no issues. Just pushed an update, thanks!